README.md

```diff
@@ -344,25 +344,42 @@ For more information about NodeJS demos go to [demo/nodejs](/demo/nodejs).
 
 ### C Demos
 
-[Microphone demo](/demo/c/porcupine_demo_mic.c) runs on Linux-based systems (e.g. Ubuntu, Raspberry Pi, and BeagleBone).
-Build the demo:
+The [Microphone demo](/demo/c/porcupine_demo_mic.c) requires `miniaudio` for accessing microphone audio data. 
+Run the following command to initialize the `miniaudio` submodule:
+
+```console
```

Comment:
good thinking. let's do it like this:

https://github.com/Picovoice/picovoice#demos